CHANGELOG.md

```diff
@@ -2,11 +2,197 @@
+    This will will now be taken into account for:
+    - `wp_strip_all_tags()` is only a valid alternative for the PHP native `strip_tags()` when the second parameter `$allowed_tags` has not been passed.
+    - `wp_parse_url()` only added support for the second parameter `$component` of the PHP native `parse_url()` function in WP 4.7.0.
+- The `WordPress.WP.DeprecatedFunctions` sniff will now detect functions deprecated in WP 4.9.
+- The `WordPress.WP.GlobalVariablesOverride` sniff will now display the name of the variable being overridden in the error message.
+- The `WordPress.WP.I18n` sniff now extends the `AbstractFunctionRestrictionSniff`.
+- Assignments in conditions in ternaries as detected by the `WordPress.CodeAnalysis.AssignmentInCondition` sniff will now be reported under a separate error code `FoundInTernaryCondition`.
+- The default error level for the notices from the `WordPress.DB.DirectDatabaseQuery` sniff has been lowered from `error` to `warning`. For the `VIP` ruleset, the error level remains `error`.
+- The default error level for the notices from the `WordPress.Security.PluginMenuSlug` sniff has been lowered from `error` to `warning`. For the `VIP` ruleset, the error level remains `error`.
+- The default error level for the notices from the `WordPress.WP.CronInterval` sniff has been lowered from `error` to `warning`. For the `VIP` ruleset, the error level remains `error`.
+- The `Sniff::get_function_call_parameters()` utility method now has improved handling of closures when passed as function call parameters.
+- Rulesets: a number of error codes were previously silenced by explicitly `exclude`-ing them. Now, they will be silenced by setting the `severity` to `0` which makes it more easily discoverable for maintainers of custom rulesets how to enable these error codes again.
+- Various performance optimizations which should most notably make a difference when running WPCS on PHP 7.
+- References to the WordPress.com VIP platform have been clarified.
+- Unit Tests: custom properties set in unit test files are reset after use.
+- Various improvements to the ruleset used by the WPCS project itself and minor code clean up related to this.
+- CI: Each change will now also be tested against the lowest supported PHPCS 3 version.
+- CI: Each change will now also be checked for PHP cross-version compatibility.
+- CI: Minor changes to the script to make the build testing faster.
+- Updated the [custom ruleset example](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/phpcs.xml.dist.sample) for the changes contained in this release and to reflect current best practices regarding the PHPCompatibility standard.
+- The instructions on how to set up WPCS for various IDEs have been moved from the `README` to the [wiki](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki).
+- Updated output examples in `README.md` and `CONTRIBUTING.md` and minor other changes to these files.
```

Comment:
The order of the adjectives in this noun phrase is unusual. minor other => other minor.